torch/_refs/nn/functional/__init__.py

```diff
@@ -59,6 +61,58 @@
 
 Tensor = torch.Tensor
 
+
+# TODO: should we allow the user to set a different dtype for the mask generation?
+def _dropout_helper(
+    self: TensorLikeType,
+    val: float,
+) -> TensorLikeType:
+
+    return refs.lt(
```

Comment:
I think this can just use the `<` operator and `torch.uniform`?

Reply:
Thanks, @mruberry! I've used the `<` operator and `torch.distributions.Uniform`.